docs/readme/benchmark.md

```diff
@@ -10,21 +10,21 @@ maxatac benchmark --prediction GM12878_CTCF_chr1.bw --gold_standard GM12878_CTCF
 
 ## Required Arguments
 
-### `--prediction`
+### `-bed, --bed, -bw, --bw, --bigwig`
```

Comment:
We do not have flags as listed here. The flags are usually --signal for a bw prediction track and different flags for bed files ("--bed", "--peaks", "--regions", "--roi", "-roi", and "--windows", "-w" for 1024 bp wide windowed BED file inputs. Please re-write, Thank you!

Reply:
Hmmm, are you sure this is the case? I am currently running benchmark successfully with the --bw option, and the current version of parser.py accepts the listed arguments (as indicated in this screenshot).
<img width="715" alt="parser_screenshot" src="https://github.com/user-attachments/assets/ffb0ac73-2d52-4d4f-b35f-c466cf24e103">